index.html

```diff
@@ -13,23 +13,39 @@
 <body>
     <!-- Accessibility check -->
     <!-- headder containing navigation links and logo -->
-
-    <header>
-        <div class="mob-nav">
-            <p class="my-logo"><a href="index.html">AlierPM</a></p>
-            <a href="#"><span class="mob-nav-menu fa fa-bars"></span></a>
-        </div>
-
-        <!-- Desktop-nav display to none here  -->
-
-        <nav class="desktop-nav">
-            <p class="my-logo"><a href="index.html">AlierPM</a></p>
-            <ul class="d-nav">
-                <li class="d-nav-link"><a href="#">Portfolio</a></li>
-                <li class="d-nav-link"><a href="#">About</a></li>
-                <li class="d-nav-link"><a href="#">Contact</a></li>
+    <header id="header">
+        <nav class="navBar" id="navbar">
+            <a href="#header" class="navLogo">AlierPM</a>
+            <hr class="rounded">
+           <ul class="navMenu leftArrow">
+              <li class="navItem">
+                <a href="#portfolio" class="navLink">Portfolio <span class="navSpan"></span></a>
+              </li>
+              <li class="navItem">
+                <a href="#about" class="navLink"
+                  >About <span class="navAbout"></span></a
+                >
+              </li>
+              <li class="navItem">
+                <a href="#contact" class="navLink">Contact <span class="navSpan"></span></a>
+              </li>
+    <li class="navItem">
+              </li>
+            </ul>
+            <ul id="nav">
+              <li class="navigation"><a href="#portfolio">Portfolio</a></li>
+              <li class="navigation"><a href="#about">About</a></li>
+              <li class="navigation"><a href="#contact">Contact</a></li>
             </ul>
-        </nav>
+            <div class="mobile-nav">
+                <a href="#header" class="mobile-logo">AlierPM</a>
+              <div class="hamburgerMenu">
+                <span class="hamburgerLine"></span>
+                <span class="hamburgerLine"></span>
+                <span class="hamburgerLine"></span>
+              </div>
+             </div>
+          </nav>
```

Comment:
- [ ] Good job adding the mobile menu 👍 but currently, the hamburger is not visible in the mobile view.   This is because you specified a margin-left of 250px which kind of takes it off the viewport as you resize. Kindly position the hamburger to match your chosen template 🤓 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/181382144-3a22e03c-a771-48cf-8884-39f3a5423fca.png)  | ![image](https://user-images.githubusercontent.com/63473584/181381644-049c49b3-f0bc-4813-ae1f-8190111e0861.png)  |